flink-connector-debezium/src/main/java/io/debezium/relational/RelationalChangeRecordEmitter.java

```diff
@@ -172,7 +164,7 @@ protected void emitUpdateRecord(Receiver<P> receiver, TableSchema tableSchema)
                     newKey,
                     envelope,
                     getOffset(),
-                    getStaticConnectHeaders());
+                    getEmitConnectHeaders().orElse(null));
```

Comment:
I don't catch here, why we use`null` instead of `new ConnectHeaders()`?

Reply:
The debezium original code uses `null` here.